src/class/api.ga4gh.php

```diff
@@ -211,6 +214,11 @@ function __construct (&$oAPI)
         // To not duplicate code.
         $this->aValueMappings['effect'] = $_SETT['var_effect_api'];
 
+        // Retrieve default genome build.
+        global $_DB;
+        $this->aActiveGBs = $_DB->query('SELECT id, column_suffix FROM ' . TABLE_GENOME_BUILDS)->fetchAllGroupAssoc();
+        $this->sActiveGB = array_keys($this->aActiveGBs)[0];
```

Comment:
Translated feedback from email @ifokkema:

>This is not available in PHP<5.4.0. Use `array_key_first()` instead.

Reply:
Yeah, so I see I didn't pay attention there - `array_key_first()` is even newer (PHP >= 7.3.0) so is not a good replacement. `key()` will probably do the trick, but should be tested. If it doesn't work, `current()` over the `array_keys()` should always work.

Reply:
`key()` did the trick :)

Reply:
Excellent!